monitoring/storage.go

```diff
@@ -101,3 +105,49 @@ func (d HighWatermarkCheckerData) SuccessMessage() string {
 
 // DiskSpaceCheckerID is the checker that checks disk space utilization
 const DiskSpaceCheckerID = "disk-space"
+
+// PathUIDCheckerData is attached to path UID check results.
+type PathUIDCheckerData struct {
```

Comment:
nit: for me, these two could be implemented as a single ID checker - would be simpler? They do not require implementation deviations beyond the semantics of the storage variable names (uid vs gid).